docs/installation/slack/index.md

```diff
@@ -5,7 +5,7 @@ sidebar_position: 1
 ---
 
 :::info
-[Legacy Slack integration](/0.14/installation/slack/) known from previous Botkube versions has been deprecated and the old Botkube Slack app will be unapproved from the Slack App Directory on the December 21st 2022. Follow the tutorial below to use the new Slack integration.
+[Legacy Slack integration](/0.14/installation/slack/) known from previous Botkube versions has been deprecated and removed from the Slack App Directory. Follow the tutorial below to use the new Slack integration.
```

Comment:
This is something we should also apply to older versions 🤔

Reply:
hm.. 🤔 I thought to not do that as for <0.17 it was true :D as they were released before 21st Dec 😄 

But I can update that in all places too.

Reply:
Essentially we can replace all occurrences of `will be unapproved from the Slack App Directory on the December 21st 2022` to `was unapproved from the Slack App Directory on the December 21st 2022` as it already happened, so... 😄